CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - Using precise dependencies (~=) in packages.json
+- Provide default value (false) for tls.rejectUnauthorized config option
```

Comment:
Is this PR related with https://github.com/telefonicaid/perseo-fe/issues/272? To mention the issue number in that case in the CNR line...

Reply:
It's a part for this issue, pending ignoreTLS/requireTLS